internal/hactar/hactar.go

```diff
@@ -39,11 +55,15 @@ func NewClient(httpClient *http.Client) *Client {
 	c.Nodes = &nodesServices{client: c}
 	c.DiskInfo = &diskInfoService{client: c}
 
+	if token != nil {
+		c.Token = util.String(token)
+	}
+
 	return c
 }
 
 func (c *Client) IsActive() bool {
-	request, err := c.NewRequest(http.MethodGet, "/health", nil)
+	request, err := c.NewRequest(http.MethodGet, healthEndpoint, nil)
```

Comment:
Not required for our backend. User can't affect on that and we suppose it's always good :)